references/cli/portforward.go

```diff
@@ -303,15 +303,30 @@ func (o *VelaPortForwardOptions) Complete() error {
 				case float64:
 					val = strconv.Itoa(int(pv))
 				default:
-					return fmt.Errorf("invalid type '%s' of port %v", reflect.TypeOf(v), k)
+					return false, fmt.Errorf("invalid type '%s' of port %v", reflect.TypeOf(v), k)
 				}
 				if val == "80" {
-					val = "8080:80"
+					val = "8088:80"
 				} else if val == "443" {
 					val = "8443:443"
 				}
 				o.Args = append(o.Args, val)
-				found = true
+				return true, nil
+			}
+			if k == "port" {
```

Comment:
I think we do not need to get a port from the `port` or `ports` fields, users must specify the port mapping.

Reply:
Users can use the command  'vela port-forward nginx 9082:80' to specify the port or the command 'Velaport-forward nginx' to establish the mapping relationship between all ports. And is is a past usage habit？ @barnettZQG

Reply:
The main reason is that this logic is not rigorous unless we are limiting some particular types of components.

Reply:
This is a convenient way, we need it

Reply:
Discover the port info from the select resource(service、deployment or pod) is a better way, currently, accept this temporary change.